views/component/recipients.js

```diff
@@ -0,0 +1,82 @@
+const { h, map } = require('mutant')
+const addSuggest = require('suggest-box')
+
+function Recipients (opts) {
+  const {
+    state,
+    myKey,
+    suggest = {},
+    avatar = (id, size) => id,
+    i18n = (key) => {
+      const translations = { 'gathering.action.addMoreRecps': '(optional) invite to this scry' }
```

Comment:
change this from gatherings.action.addMoreRecps to something scry related

Reply:
renamed to `scry.action.invite`